src/nvim/normal.c

```diff
@@ -397,6 +397,11 @@ void init_normal_cmds(void)
   nv_max_linear = i - 1;
 }
 
+void init_default_mappings(void)
+{
+  add_map((char_u *)"Y y$", NORMAL | VISUAL);
+}
+
```

Comment:
@justinmk I know you moved this from `main.c` to `normal.c`, but for non-normal mode default mappings (e.g. `<C-U>` in insert mode) does it make sense to move this back into `main.c` or should we just put all defaults here in `normal.c`?

Reply:
i guess technically it should go in getchar.c